spec/app/ics-028-cross-chain-validation/system_model_and_properties.md

```diff
@@ -169,6 +204,28 @@ The following properties define the guarantees of CCV on *registering* on the pr
 
 > Note that, except for *Apply VSC Liveness* and *Register Maturity Liveness*, none of the properties of CCV require the *Correct Relayer* assumption to hold.
 
+### Consumer Initiated Slashing
+[&uparrow; Back to Outline](#outline)
+
+- ***Consumer Slashing Warranty***: Let `cc` be a consumer chain, such that its CCV module receives at height `he` evidence that a validator `val` misbehaved on `cc` at height `hi`. 
+  Let `hv` be the height when the CCV module of `cc` receives the first VSC from the provider CCV module, i.e., the height when the CCV channel is established.
+  Then, the CCV module of `cc` MUST send (to the provider CCV module) *exactly one* `SlashPacket` `P`, such that
+    - `P` is sent at height `h = max(he, hv)`;
+    - `P.val = val` and `P.id = HtoVSC[hi]`,
+    i.e., the ID of the latest VSC that updated the validator set on `cc` at height `hi` or `0` if such a VSC does not exist (if `hi < hv`).
+  > **Note**: A consequence of the *Consumer Slashing Warranty* property is that the initial validator set on a consumer chain cannot be slashed during the initialization of the CCV channel. 
+  Therefore, consumer chains *SHOULD NOT allow user transactions before the CCV channel is established*. 
+  Note that once the CCV channel is established (i.e., a VSC is received from the provider CCV module), CCV enables the slashing of the initial validator set for infractions committed during channel initialization.
+
+- ***Provider Slashing Warranty***: If the provider CCV module receives in a block from a consumer chain `cc` a `SlashPacket` containing a validator `val` and a VSC ID `vscId`, 
```

Comment:
I am not sure about the wording "in a block" in this property. Above "at height" is used which seems clearer to me.